chacha20/Cargo.toml

```diff
@@ -5,11 +5,12 @@ authors = ["RustCrypto Developers"]
 license = "MIT OR Apache-2.0"
 description = "Chacha20 Stream Cipher"
 repository = "https://github.com/RustCrypto/stream-ciphers"
-keywords = ["crypto", "stream-cipher", "trait"]
+keywords = ["chacha20poly1305", "crypto", "hchacha20", "stream-cipher", "xchacha20"]
```

Comment:
Are you sure about `chacha20poly1305` flag? Or do you intend to add it later?

Reply:
Good point... we should just ship a `chacha20poly1305` crate whenever https://github.com/RustCrypto/traits/pull/40 lands